### PR TITLE
Add 1.9.12 mainnet release

### PR DESCRIPTION
**Pull request by WilfredTA**

### CODE REVIEW

File: vercel.json [hunk 1]
```diff
@@ -2,12 +2,12 @@
   "redirects": [
     {
       "source": "/releases/mainnet/active/contracts.json",
-      "destination": "/releases/mainnet/1.9.11/contracts.json",
+      "destination": "/releases/mainnet/1.9.12/contracts.json",
```
Comment (naszam) [inline vercel.json:5]: I think we should wait the execution of the spell before updating active links

File: releases/mainnet/1.9.12/index.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+			<h2>
+				<strong>Mainnet Release 1.9.12</strong>
+			</h2>
+
+			<ul>
+				<li><a href="/releases/mainnet/1.9.12/contracts.json">Contract addresses</a></li>
+				<li><a href="https://ipfs.io/ipfs/QmYojffjj3t9ifkkyATD9xr2zPMPsvpMgcoqzVtppAhNKQ">Contract addresses (IPFS hosted)</a></li>
+				<li><a href="/releases/mainnet/1.9.12/abi/index.html">ABIs</a></li>
+
+				<li>Links to GitHub repositories
```
Comment (naszam) [inline releases/mainnet/1.9.12/index.html:16]: to check with @gbalabasquer about that, but I think we could keep it commented for now

File: releases/mainnet/1.9.12/contracts.json [hunk 3]
```diff
@@ -0,0 +1,297 @@
...
+    "ARBITRUM_ESCROW": "0xA10c7CE4b876998858b1a9E12b10092229539400",
+    "ARBITRUM_GOV_RELAY": "0x9ba25c289e351779E0D481Ba37489317c34A899d",
+    "GUNIV3DAIUSDC2": "0x50379f632ca68D36E50cfBC8F78fe16bd1499d1e",
+    "PIP_GUNIV3DAIUSDC2": "0xcCBa43231aC6eceBd1278B90c3a44711a00F4e93",
+    "MCD_JOIN_GUNIV3DAIUSDC2_A": "0xA7e4dDde3cBcEf122851A7C8F7A55f23c0Daf335",
+    "MCD_CLIP_GUNIV3DAIUSDC2_A": "0xB55da3d3100C4eBF9De755b6DdC24BF209f6cc06",
+    "MCD_CLIP_CALC_GUNIV3DAIUSDC2_A": "0xef051Ca2A2d809ba47ee0FC8caaEd06E3D832225"
+  
+  }
+
```
Comment (naszam) [inline releases/mainnet/1.9.12/contracts.json:297]: extra line - eof missing newline

File: releases/mainnet/1.9.12/abi/index.html [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<html>
+	<div>
+		<h2>
+			<strong>Mainnet Release 1.9.12 - ABIs</strong>
+		</h2>
+		<p>Download an <a href="/releases/mainnet/1.9.11/abi/mainnet_abi_1.9.12.zip" target="_blank">archive of all Mainnet 1.9.11 ABIs</a>.<p>
```
Comment (naszam) [inline releases/mainnet/1.9.12/abi/index.html:6]:
```suggestion
		<p>Download an <a href="/releases/mainnet/1.9.12/abi/mainnet_abi_1.9.12.zip" target="_blank">archive of all Mainnet 1.9.12 ABIs</a>.<p>
```

File: releases/mainnet/1.9.12/contracts.json (hunk 3 above)
Comment (naszam) [reply releases/mainnet/1.9.12/contracts.json:297]: some spacing as well ([diffcheck](https://www.diffchecker.com/l7fD34QT))

Comment (naszam) [reply releases/mainnet/1.9.12/contracts.json:297]: we should hash it again reflecting the changes and update the `index.html` as well

File: vercel.json (hunk 1 above)
Comment (gbalabasquer) [reply vercel.json:5]: Yes if, this gets merged before the execution of the spell, then the active link should remain to 1.9.11

File: releases/mainnet/1.9.12/index.html (hunk 2 above)
Comment (gbalabasquer) [reply releases/mainnet/1.9.12/index.html:16]: Leave it uncommented. I'll get `dss-deploy-scripts` 1.9.12 soon.
Btw I'm realizing I forgot to push 1.9.11, will do that soon as well.

File: releases/mainnet/1.9.12/contracts.json [hunk 5]
```diff
@@ -0,0 +1,297 @@
...
+    "RWA006_A_OUTPUT_CONDUIT": "0x8Fe38D1E4293181273E2e323e4c16e0D1d4861e3",
+    "PROXY_PAUSE_ACTIONS": "0x6bda13D43B7EDd6CAfE1f70fB98b5d40f61A1370",
+    "PROXY_DEPLOYER": "0x1b93556AB8dcCEF01Cd7823C617a6d340f53Fb58",
+    "OPTIMISM_DAI_BRIDGE": "0x10E6593CDda8c58a1d0f14C5164B376352a55f2F",
+    "OPTIMISM_ESCROW": "0x467194771dAe2967Aef3ECbEDD3Bf9a310C76C65",
+    "OPTIMISM_GOV_RELAY": "0x09B354CDA89203BB7B3131CC728dFa06ab09Ae2F",
+    "ARBITRUM_DAI_BRIDGE": "0xD3B5b60020504bc3489D6949d545893982BA3011",
+    "ARBITRUM_ESCROW": "0xA10c7CE4b876998858b1a9E12b10092229539400",
+    "ARBITRUM_GOV_RELAY": "0x9ba25c289e351779E0D481Ba37489317c34A899d",
+    "GUNIV3DAIUSDC2": "0x50379f632ca68D36E50cfBC8F78fe16bd1499d1e",
```
[the remark this answers is not in the document]
Comment (gbalabasquer) [reply releases/mainnet/1.9.12/contracts.json:290]: Yeah after the other GUNI looks good. They are different tokens so explicit rule, but we have done some grouping for regular tokens, UNIV2 and RWAs. I'd say not definitely not after the arbitrum contracts.

File: releases/mainnet/1.9.12/index.html (hunk 2 above)
Comment (gbalabasquer) [reply releases/mainnet/1.9.12/index.html:16]: 1.9.11 and 1.9.12 releases added to `dss-deploy-scripts`.

**Review by naszam — APPROVED** [review]
Checked:

- Active Links Updated (1.9.12)
- `contracts.json` Looks Good ([diffcheck](https://www.diffchecker.com/3WMHmhEP) against 1.9.11)
- IPFS Hash Matches `contracts.json` ([diffcheck](https://www.diffchecker.com/mgViwvlF))
- Changelog Changes Look Good (`GUNIV3DAIUSDC2`)
- ABIs Looks Good (Match 1.9.11, No New ABIs)

Note: if the pr gets merged before [execution of spell](https://vote.makerdao.com/executive?netw), active link should remain to 1.9.11

LGTM

**Review by zdumitru — APPROVED** [review]
Approved. To be merged only AFTER the spell is cast, since 'active' links are updated here in the pr.

**Pull request merged**